### PR TITLE
com_fields - Fix deprecated warning in group model

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue # .

### Summary of Changes
Fix warning 
Deprecated: substr(): Passing null to parameter #1 ($string) of type string is deprecated in administrator\components\com_fields\src\Model\GroupModel.php on line 323

### Testing Instructions
Add a new field group in com_contacts (php 8.1.5)

### Actual result BEFORE applying this Pull Request
grafik

### Expected result AFTER applying this Pull Request
No deprecated warning

### CODE REVIEW

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on c94011d33791eb4c9f1893d0659fccdd654ccc47<br /><br />Tested successfully in Joomla 4.1.5-dev of 1 June in Wampserver 3.2.9 using PHP 8.1.6.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37932">issues.joomla.org/tracker/joomla-cms/37932</a>.</sub>

File: administrator/components/com_fields/src/Model/GroupModel.php [hunk 1]
```diff
@@ -320,7 +320,9 @@ protected function loadFormData()
 			if (!$data->id)
 			{
 				// Check for which context the Field Group Manager is used and get selected fields
-				$context = substr($app->getUserState('com_fields.groups.filter.context'), 4);
```
Comment (alikon) [inline administrator/components/com_fields/src/Model/GroupModel.php:323]: i think should be better
```php
$context = substr($app->getUserState('com_fields.groups.filter.context', ''), 4);
```

**Review by alikon — CHANGES_REQUESTED** [review]

File: administrator/components/com_fields/src/Model/GroupModel.php (hunk 1 above)
Comment (chmst) [reply administrator/components/com_fields/src/Model/GroupModel.php:323]: thx

**Comment by chmst** [comment]
@toivo could you please re-test?

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on df2a42d4f06c5261876cef2fa642d49e1e19db32<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37932">issues.joomla.org/tracker/joomla-cms/37932</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on df2a42d4f06c5261876cef2fa642d49e1e19db32<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37932">issues.joomla.org/tracker/joomla-cms/37932</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37932">issues.joomla.org/tracker/joomla-cms/37932</a>.</sub>

**Review by richard67 — APPROVED** [review]

**Comment by bembelimen** [comment]
Thx

**Pull request merged**